src/internal/sagaHelpers.js

```diff
@@ -94,3 +95,11 @@ export function throttle(delayLength, pattern, worker, ...args) {
     q4() { return ['q2', yDelay] }
   }, 'q1', `throttle(${safeName(pattern)}, ${worker.name})`)
 }
+
+const deprecationWarning = (helperName) =>
+`import ${helperName} from 'redux-saga' got deprecated in favor of import ${helperName} from 'redux-saga/effects'.
```

Comment:
I had just noticed this was inconsistent with the `put.sync` deprecation warning above
```js
put.sync = deprecate(put.resolve, 'put.sync is deprecated in favor of put.resolve, please update your code')
```